docs/CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ The format is based on [Keep a Changelog][kc], and this project adheres to
 * `PrefabData` can now be derived for many situations, see the book for more information ([#1035])
 * Support for DirectionalLight and SpotLight in PBM pass. ([#1074], [#1081])
 * `amethyst_ui::get_default_font` supports loading system font from Path. ([#1108])
+* `sprite_camera_follow` example showing how to use a Camera that has a sprite Parent ([#919])
```

Comment:
You forgot to add the link for this changelog entry.

```rs
[#1099]: https://github.com/amethyst/amethyst/pull/1099
```

Reply:
Ah, I misunderstood how those links worked. Fixed.